routemaster/context.py

```diff
@@ -11,14 +13,17 @@ class Context(object):
 
     def __init__(
         self,
+        *,
         label: str,
-        metadata: Dict[str, Any],
-        now: datetime.datetime,
-        feeds: Dict[str, Feed],
-        accessed_variables: Iterable[str],
+        metadata: Dict[str, Any] = {},
+        now: datetime.datetime = None,
+        feeds: Dict[str, Feed] = {},
+        accessed_variables: Iterable[str] = [],
```

Comment:
I'm not sure that having these mutable defaults is a good idea (maybe add `flake8-mutable`?).

Perhaps a better way to solve this would be a helper used in tests which handles the non-interesting defaults?

Reply:
I've fixed these with a test factory method that has the defaults (but not as mutable args).